README.md

```diff
@@ -23,11 +23,11 @@ It can connect to multiple engines at the same time and provides an unified inte
 
 Specific runtime plug-ins provides support for different container engine.
 
-There are plug-ins to handle [Podman container engine](https://github.com/containers/podman), [CodeReady container](https://github.com/code-ready/crc), [Podman Lima machines](https://github.com/lima-vm/lima), Docker, etc.
+There are plug-ins to handle [Podman container engine](https://github.com/containers/podman), [crc](https://github.com/code-ready/crc), [Podman Lima machines](https://github.com/lima-vm/lima), Docker, etc.

```

Comment:
I would say we would need the 'full name' and not the acronym for the name.

Reply:
the new name is "OpenShift Local" for the downstream, but only 'crc', with no direct meaning for upstream.

Reply:
but there https://github.com/code-ready/crc it's written `Container Runtime Control` and `CodeReady Containers` everywhere

Reply:
that will change in the next weeks; (name change, and the removal of the CodeReady brand)